test/handler.test.js

```diff
@@ -45,6 +47,7 @@ describe('HTTP request handler', () => {
   beforeEach(async () => {
     await pgPool.query('DELETE FROM retrieval_stats')
     await pgPool.query('DELETE FROM daily_participants')
+    await pgPool.query('DELETE FROM daily_node_metrics')
```

Comment:
I think this cleanup should go to `test/platform-routes.test.js` only, as the tests in this file are not creating any new rows in `daily_node_metrics`. WDYT?

Reply:
Ah yep! I have it in platform-routes.test.js, just forgot to remove it here